index.html

```diff
@@ -0,0 +1,462 @@
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <header class="up-side">
+      <p class="logo">My Logo</p>
+      <nav class="union">
+        <img src="icons/Union.png" />
+      </nav>
+      <nav class="buttons-up">
+        <ul>
+          <li class="li-up"><a href="#work">Portfolio</a></li>
+          <li class="li-up"><a href="#about">About</a></li>
+          <li class="li-up"><a href="#contact-form">Contact</a></li>
+          <li class="li-up">
+            <a href="mailto:meryemsanemkucukbayram@gmail.com">
+              <img src="icons/mail-icon.png" />
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <section class="headline">
+      <h1 class="alex">
+        Hey There. <br />
+        I'm Alex.
+      </h1>
+      <h3 class="soft">I'm a Software Developer</h3>
+      <p class="text">
+        I can help you build a product , feature or website Look through some of
+        my work and experience! If you like what you see and have a project you
+        need coded, don’t hesitate to contact me.
+      </p>
+      <ul class="icon">
+        <li class="icon1">
+          <a href="#"><img src="icons/icon1.png" /></a>
+        </li>
+        <li class="icon2">
+          <a href="#"><img src="icons/icon2.png" /></a>
+        </li>
+        <li class="icon3">
+          <a href="#"><img src="icons/icon3.png" /></a>
+        </li>
+        <li class="icon4">
+          <a href="#"><img src="icons/icon4.png" /></a>
+        </li>
+        <li class="icon5">
+          <a href="#"><img src="icons/icon5.png" /></a>
+        </li>
+      </ul>
+    </section>
+
+    <div class="grid-headline">
+      <section class="headline2">
+        <h1 class="alex">
+          Hey There. <br />
+          I'm Alex.
+        </h1>
+        <h3 class="soft">I'm a Software Developer</h3>
+        <p class="text">
+          I can help you build a product , feature or website Look through some
+          of my work and experience! If you like what you see and have a project
+          you need coded, don’t hesitate to contact me.
+        </p>
+        <ul class="icon-desk">
+          <li class="icon1">
+            <a href="#"><img src="icons/icon1.png" /></a>
+          </li>
+          <li class="icon2">
+            <a href="#"><img src="icons/icon2.png" /></a>
+          </li>
+          <li class="icon3">
+            <a href="#"><img src="icons/icon3.png" /></a>
+          </li>
+          <li class="icon4">
+            <a href="#"><img src="icons/icon4.png" /></a>
+          </li>
+          <li class="icon5">
+            <a href="#"><img src="icons/icon5.png" /></a>
+          </li>
+        </ul>
+        <section class="image-head">
+          <img src="icons/desktop-ill1.png" />
+        </section>
```

Comment:
- [ ] As required for the project to sick as much as possible to the your chosen Figma template design and for best practices, kindly ensure that you fix the overlapping issue the your background image. This happens because you use this image in `img` tag instead of using it as a css background image as it was required in your first milestone. Look below how looks your design(you can fix it by removing this image in your htmtl file and then use it as the css background of your Headline section) 
![Screenshot from 2023-05-12 13-25-19](https://github.com/meryemsanem/desktop-version/assets/83042742/74d965f2-ba80-4232-80e9-23c746e02569)